setup.py

```diff
@@ -104,7 +104,8 @@
             "dev": doc_requirements
             + test_requirements
             + interactive_requirements
-            + modin_requirements,
+            + modin_requirements
+            + ["ray", "dask"],
```

Comment:
Shouldn't these be added to the `modin_requirements` list?

Reply:
Modin has multiple backends (more even than these two) and they are all optional, since you can install any _one_ of them and it will work. We don't want to force our users to install any particular backend, but for development I've been using these two (and I think we will test on these two, once we have automated tests for Modin).

Reply:
We shouldn't have naked requirements declared here. Sounds like we need a separate variable to contain these requirements like `modin_test_requirements`. If we have automated tests for these, they should at least be within the "test" extra otherwise tests will fail if you only install `.[test]`.

Reply:
@rmudambi I've updated this; let me know if it is better now. We don't actually have the automated tests yet, but we will want them before we merge the Modin epic.